core/meterer/meterer_test.go

```diff
@@ -96,12 +105,15 @@ func setup(_ *testing.M) {
 	}
 
 	logger = logging.NewNoopLogger()
+	signer = auth.NewEIP712Signer(big.NewInt(17000), gethcommon.HexToAddress("0x1234000000000000000000000000000000000000"))
 	config := meterer.Config{
-		PricePerChargeable:   1,
-		MinChargeableSize:    1,
-		GlobalBytesPerSecond: 1000,
-		ReservationWindow:    60,
-		ChainReadTimeout:     3 * time.Second,
+		PricePerSymbol:         1,
```

Comment:
Would it be worth using some unique prime values here for different parameters (e.g. 2,3,5,7) to give better confidence that the calcualtions are being done correctly?